.github/workflows/update-base.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     if: ${{ github.event.label.name == 'update-base' }}
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/github-script@v3
+      - uses: actions/github-script@v4
```

Comment:
I assume this didnt break anything? One way to find out I guess

Reply:
Guess it didn't